pydm/widgets/baseplot.py

```diff
@@ -606,6 +619,10 @@ def maxRedrawRate(self, redraw_rate):
         self._redraw_rate = redraw_rate
         self.redraw_timer.setInterval(int((1.0/self._redraw_rate)*1000))
 
+    def pausePlotting(self):
```

Comment:
We should rename this to something like: `toggle_plot_update` since it not necessarily just stops the plot.

Reply:
Also, missing docstrings